dm/pkg/checker/mysql_server.go

```diff
@@ -35,14 +35,14 @@ func NewMySQLVersionChecker(db *sql.DB, dbinfo *dbutil.DBConfig) RealChecker {
 }
 
 // SupportedVersion defines the MySQL/MariaDB version that DM/syncer supports
-// * 5.6.0 <= MySQL Version < 8.0.0.
+// * 5.6.0 <= MySQL Version < 8.0.34.
 var SupportedVersion = map[string]struct {
 	Min MySQLVersion
 	Max MySQLVersion
 }{
 	"mysql": {
 		MySQLVersion{5, 6, 0},
-		MySQLVersion{8, 0, 0},
+		MySQLVersion{8, 0, 34},
```

Comment:
why do we choose this version?

https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-35.html

Reply:
right, supported version should be < `8.0.36`

Reply:
can you explain the reason why we choose that version? currently latest MySQL's release version?

Reply:
yes, I initially tend to support `8.0.x`, which means we support all the versions starting with `8.0`, but in the code we need to be specific about the version, so I use the latest MySQL's release version

Reply:
You can use `MySQLVersion{8, 1, 0}` then

Reply:
modified